docs/content/Deployment/Platform-Docker.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Deploy with Docker
+permalink: /deployment/platforms/docker
+category: Deployment
+subCategory: Platforms
+menuOrder: 1
+---
+
+This guide walks you through deploying Cube.js with Docker.
+
+## Prerequisites
+
+- Docker CLI
```

Comment:
Having prerequisites is really cool. It also great to provide links to install or set up the prerequisites. (It also seems that there's no clear way to install "Docker CLI", just "[Docker Desktop](https://www.docker.com/products/docker-app)".)